src/pages/campaigns/components/campaignForm/index.tsx

```diff
@@ -53,6 +55,40 @@ const StickyContainer = styled.div`
   }
 `;
 
+const SurveyButton = ({ campaign_id }: { campaign_id: string }) => {
+  const { data } = useGetCampaignsFormsQuery({
+    searchBy: "campaign_id",
+    search: campaign_id,
+  });
+
+  const path = useLocation().pathname;
+  if (path === "/backoffice/campaigns/new") return null;
```

Comment:
non dovrebbe servire perché il controllo se sei in edit lo fai già a monte nella index.tsx, cioè se gli passi un dossier.id

Reply:
rimosso. effettivamente era in più!